paparazzi/src/test/java/app/cash/paparazzi/PaparazziTest.kt

```diff
@@ -108,12 +108,16 @@ class PaparazziTest {
       "onDraw time=1000 animationElapsed=0.0",
       "onAnimationStart time=2000 animationElapsed=0.0",
       "onAnimationUpdate time=2000 animationElapsed=0.0",
+      "onAnimationUpdate time=2000 animationElapsed=0.0",
```

Comment:
A side-effect of `Choreographer_Delegate.doFrame` and `choreographerCallbacks.execute(currentTimeNanos, Bridge.getLog())` executing multiple update callback.

Need to look further into this as removing `choreographerCallbacks.execute(currentTimeNanos, Bridge.getLog())` causes frame times for animations to be incorrect